test/docfx.Test/docfx.test.yml

```diff
@@ -1,4 +1,5 @@
 # default config for specs in docfx
+baseUrl: https://docs.com
```

Comment:
Feels confused as `docs.com` is use both as baseUrl and in `repos` in spec tests. I think it is more suitable here, so we can use another fake site for `repos`.